### PR TITLE
src/clientHandler: add support for certain CLI options

### PR DESCRIPTION
**Pull request by danishprakash**
* terraformExecPath
* terraformExecTimeout
* terraformExecLogFilePath

Closes (partially) https://github.com/hashicorp/terraform-ls/issues/240

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -149,6 +149,24 @@
           "default": [],
           "description": "Per-workspace list of module directories for the language server to read"
         },
+        "terraform-ls.terraformExecPath": {
+          "scope": "resource",
+          "type": "string",
+          "default": [],
+          "description": "Path to Terraform binary"
```
Comment (radeksimko) [inline package.json:156]:
```suggestion
          "description": "Path to the Terraform binary"
```

File: package.json [hunk 2]
```diff
@@ -149,6 +149,24 @@
...
+          "scope": "resource",
+          "type": "string",
+          "default": [],
+          "description": "Overrides Terraform execution timeout (e.g. 30s)"
+        },
+        "terraform-ls.terraformExecLogFilePath": {
+          "scope": "resource",
+          "type": "string",
+          "default": [],
+          "description": "Path to a file for Terraform executions to be logged into with support for variables (e.g. Timestamp, Pid, Ppid) via Go template syntax {{.VarName}}"
```
Comment (radeksimko) [inline package.json:168]:
```suggestion
          "description": "Path to a file for Terraform executions to be logged into (TF_LOG_PATH) with support for variables (e.g. Timestamp, Pid, Ppid) via Go template syntax {{.VarName}}"
```

File: package.json [hunk 3]
```diff
@@ -149,6 +149,24 @@
...
+        "terraform-ls.terraformExecTimeout": {
+          "scope": "resource",
+          "type": "string",
+          "default": [],
+          "description": "Overrides Terraform execution timeout (e.g. 30s)"
+        },
+        "terraform-ls.terraformExecLogFilePath": {
+          "scope": "resource",
+          "type": "string",
+          "default": [],
```
Comment (radeksimko) [inline package.json:167]: The default type seems wrong (array) for string arguments but more importantly I don't think we need any defaults for these options in particular? Do you mind removing the defaults?

Comment (danishprakash) [reply package.json:167]: You're right, didn't notice that while copying over this block from an existing piece. Updated.

**Review by radeksimko — APPROVED** [review]
LGTM, thanks.

**Pull request merged**